### DISCUSSION TITLE
Error loading the tilemap from ogmo

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2445] [Q&A] [answered]
- **Haxe version:** i guess latest
- **Flixel version:** latest
- **OpenFL version:** latest
- **Lime version:** latest
- **Affected targets:** windows

________________________________

official guide for flixel. step 6. when I type the:
![image](https://user-images.githubusercontent.com/60806448/141133327-c0fb85ca-fc6e-48c8-8681-f36a68ff0271.png)
it tells me that it's unknown identifier even though i added the
![image](https://user-images.githubusercontent.com/60806448/141133062-028e78a3-9bae-473d-9180-86546d935d5b.png)
in Project.xml
________________________________

any help?

Edit: the FlxTilemap actually works (it suggested me to import some library and it's no error now)

**Comment by NikRetaNCAM** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/60806448/141141784-44dd4959-5c87-488c-a108-8055b0ec5617.png)
is this how it's supposed to look??
it works now
also pls if any other people had this mistake can you tell y that happened and maybe devs would actually update the tutorial to specify that..?

**Reply by Geokureli** [reply]
> also pls if any other people had this mistake can you tell y that happened and maybe devs would actually update the tutorial to specify that..?

The tutorial covers imports in step 2, not sure how you got the rest of it working without discovering imports, since every class you use needs to be imported. Also note that if you copy paste code it wont automatically add the import, but - as it says below - if you use auto complete to type the class it will automatically add the import.

<img width="821" alt="Screen Shot 2021-11-11 at 12 27 46 PM" src="https://user-images.githubusercontent.com/2609513/141350165-21168c09-58b7-4ab7-b983-28412b4920a1.png">
